beacon-chain/verification/initializer.go

```diff
@@ -57,6 +58,16 @@ func (ini *Initializer) NewBlobVerifier(b blocks.ROBlob, reqs []Requirement) *RO
 	}
 }
 
+// NewPayloadAttestationVerifier creates a PayloadAttestationVerifier for a single payload attestation message,
+// with the given set of requirements.
+func (ini *Initializer) NewPayloadAttestationVerifier(pa payloadattestation.ROMessage, reqs []Requirement) *PayloadAttVerifier {
+	return &PayloadAttVerifier{
+		sharedResources: ini.shared,
+		results:         newResults(reqs...),
+		pa:              pa,
+	}
+}
+
```

Comment:
Can we move new structures to new files, like in /verification/initializer_epbs.go? so that we avoid conflicts on rebases.